src/main/java/com/revature/services/BatchLogic.java

```diff
@@ -1,19 +1,20 @@
 package com.revature.services;
 
-import org.springframework.data.domain.Page;
-import org.springframework.data.domain.Pageable;
-
 import com.revature.domain.Batch;
 import com.revature.domain.Person;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 
 public interface BatchLogic {
 	
 	Batch getBatchByName(String batchName);
 	Batch getBatchById(int batchId);
-	void deleteBatch(Batch batchName);
+	String deleteBatch(Integer id);
+	Batch removePersonsFromBatch(Integer batchId, Integer[] personIds);
 	Page<Batch> getAllBatches(Pageable pageable);
-	void createBatch(Batch batchName);
-	void updateBatch(Batch batchName);
+	Batch createBatch(Batch batchName);
+	Batch addPersonsToBatch(Integer batchId, Integer[] personIds);
+	Batch updateBatch(Batch batchName, Integer id);
 	Page<Person> getAllPeopleByBatchId(Pageable pageable, Integer Id);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS00117)